ddtrace/contrib/structlog/patch.py

```diff
@@ -50,32 +47,34 @@ def _tracer_injection(_, __, event_dict):
     return event_dict
 
 
-def _w_configure(func, instance, args, kwargs):
+def _w_get_logger(func, instance, args, kwargs):
     dd_processor = [_tracer_injection]
-    # Only inject values if there is some sort of pre-existing processing with a valid renderer.
-    # Without this assumption, the logs will not format correctly since it can't accept the injected values
-    # This way we do not change their format and mess up any existing logs
-    arg_processors = get_argument_value(args, kwargs, 0, "processors", True)
-    if arg_processors and len(arg_processors) != 0:
-        set_argument_value(args, kwargs, 0, "processors", dd_processor + arg_processors)
-
+    structlog._config._CONFIG.default_processors = dd_processor + structlog._config._CONFIG.default_processors[:]
```

Comment:
```suggestion
    structlog._config._CONFIG.default_processors = dd_processor + structlog._config._CONFIG.default_processors
```
can't you just do this?